### ISSUE TITLE
"OWC offering MUST include a styleSet reference, referencing an external SLD file or an SLD from the owc_style table."

### ISSUE THREAD

**Issue by doublebyte1** [GeoCat/qgpkg#13]
Do we want to support this, reading an SLD from an external file? Please note that in that case the GeoPackage wont be self-contained.

**Comment by pvgenuchten** [comment]
true, users would need to be aware that when the insert an external link, the package won't be self contained anymore... maybe recommend to embed it?
